.github/workflows/build-web-extension.yml

```diff
@@ -54,3 +58,20 @@ jobs:
                       !packages/extension/dist/web-ext-artifacts
                       !packages/extension/dist/*.xpi
                   if-no-files-found: error
+            - name: Build ZIP for signing Chrome Extension
+              uses: cardinalby/webext-buildtools-pack-extension-dir-action@v1
+              with:
+                  extensionDir: packages/extension/dist
+                  zipFilePath: packages/extension/chrome.zip
+                  zipIgnore: web-ext-artifacts/**|*.xpi
```

Comment:
Unfortunately building an artifact doesn't allow us to reuse the zip (it's created in a container), so we need to do it "again".